spec/unit/jobs/enqueuer_spec.rb

```diff
@@ -117,6 +128,18 @@ module VCAP::CloudController::Jobs
           end
         end
       end
+
+      context 'priority from config' do
+        let(:job_display_name) { { wrapped_job.display_name.to_sym => 1899 } }
```

Comment:
And this one to: `let(:priorities) { { priorities: { wrapped_job.display_name.to_sym => 1899 } } }`